tests/errors/test_str_to_int.py

```diff
@@ -0,0 +1,4 @@
+def test_e1():
+    print(int('3abc'))
```

Comment:
Please also add a separate file under `integration_tests`. And make this work too:
```python
assert int("-423") == -423
```

Reply:
Thanks for pointing this case out !